code/obj/machinery/secscanner.dm

```diff
@@ -61,6 +61,8 @@
 			src.use_power(15)
 
 			//////PDA NOTIFY/////
+			if (src.loc.z != Z_LEVEL_STATION)
+				return
```

Comment:
you're ignoring the scanner icon being set back, if this was tested you would've noticed 👿

Reply:
I did this miss but it was tested. Had to find a z level scanner (Owlery, Staff Hall via jump), PDA that showed alerts (sec), make sure my ID didn't have contraband, etc.

I worked out why it's not sticking as red for an item failure, this thing constantly changes icon_state in process(), the change back just makes the timing slightly more consistent.